javascript/index.js

```diff
@@ -0,0 +1,316 @@
+
+var stack = [];
+
```

Comment:
Function countNrDecimals is used in the code only to check if a number (num) contains the decimal point.
So only the flag is important, nrDec can be deleted and the function can be simplified and made more efficient!